bin/console

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env php
 <?php
 
+declare(strict_types=1);
+
 /** @var Nette\DI\Container $container */
-$container = require __DIR__ . '/../app/bootstrap.php';
+require __DIR__ . '/../vendor/autoload.php';
```

Comment:
tohle asi neni nutne? ... a spis bych to dal pak nekdy jako samostatny commit

Reply:
Nešly mi bez toho spustit migrace.

Reply:
To chapu, ale logicky to patri do commitu, kterej opravuje fungovani spousteni migraci. Proto bych to sem nedaval a pak poslal samostatnej PR na opravu conosle.

Reply:
OK